Server/docker-compose.yml

```diff
@@ -26,24 +26,57 @@ services:
       POSTGRES_USER: citrine
       POSTGRES_PASSWORD: "citrine"
     healthcheck:
-      test: ["CMD-SHELL", "pg_isready", "-d", "db_prod"]
+      test: [ "CMD-SHELL", "pg_isready", "-d", "db_prod" ]
       interval: 30s
       timeout: 60s
       retries: 5
-      start_period: 80s  
+      start_period: 80s
   redis:
     image: redis:latest
     ports:
       - "6379:6379"
     healthcheck:
-      test: ["CMD", "redis-cli", "ping"]
+      test: [ "CMD", "redis-cli", "ping" ]
       interval: 10s
       timeout: 5s
       retries: 3
   citrine:
     build:
-      context: ../
-      dockerfile: ./Server/docker/Dockerfile
+      context: ..
+      dockerfile: ./Server/local.Dockerfile
+    volumes:
+      - ../:/usr/local/apps/citrineos
+      - /usr/local/apps/citrineos/node_modules
```

Comment:
why are these volumes being added?

Reply:
Makes sense! Please add this improvement to Swarm as well.

Reply:
Yeah I was thinking about doing this but left it as though the Swarm docker setup would be used as a "deployed" environment and not something that you would really run locally. I can agree that it would also make sense to have a Swarm version that has the same volumes added, but then don't we also need another file added that does not mount the volumes for cases where you just want to run the setup in a deployed environment?

Reply:
I am going to see how we can do this easily because I really don't wish to copy that long list everywhere, it is really meant more for ease of development purposes, but if we do want to run swarm locally in dev as we may really want to do, then we need this in swarm for sure as well

Reply:
There are other considerations for Swarm, for sure. I can see leaving this out of it for now. Please add back the Swarm folder. We can discuss how best to carry forward the intent behind Swarm in future PRs. [I have created a discussion topic for it.](https://github.com/orgs/citrineos/discussions/19)